web/packages/core/src/ruffle-player.ts

```diff
@@ -220,8 +222,14 @@ export class RufflePlayer extends HTMLElement {
         this.preloader = this.shadow.getElementById("preloader")!;
 
         this.contextMenuElement = this.shadow.getElementById("context-menu")!;
-        this.addEventListener("contextmenu", this.showContextMenu.bind(this));
+        this.addEventListener("contextmenu", this.noContextMenu.bind(this));
```

Comment:
Wouldn't this break context menu on desktop (where `contextmenu` is triggered by a right-click)?

Reply:
No because pointer events cover mouse events and touch events. See the code near your other comment for how mouse pointer events are handled. The "contextmenu" event is blocked here simply to prevent the default browser context menu from appearing.

Reply:
Got it, but I still feel like using the `contextmenu` event is preferred when available, to avoid simulating its (not no complex, but still) logic ourselves. This can be accomplished by skipping the added functionality in the `pointerdown` handler if the `contextmenu` event is available, which can be checked via:
```js
"oncontextmenu" in window
```

Reply:
It's worth noting that Flash Player's context menu behavior is closer to this behavior than that of the default `contextmenu` event. Flash Player's context menu appears on rightMouseButtonDown, while the `contextmenu` event is fired on rightMouseButtonUp.

Reply:
Also, I just checked, and `"oncontextmenu" in window` returns `true` on my iPhone.

Reply:
> "oncontextmenu" in window returns true on iPhone.

This is the issue and the reason I can't use feature detection even though I wanted to too. I asked Adrian to check on discord and he found that this was the case. It's because the context menu event can fire when you plug a Bluetooth mouse into an iPhone. Just not with long press.

Reply:
It's also arguably better to use the same or similar logic for both desktop and phones. Also the noContextMenu function is because without this function on desktop right click would show the default browser context menu above the Ruffle context menu.

Reply:
nosamu added the same type of feature detection isTouch has.